Sources/Gui/MainScreenHelper.h

```diff
@@ -59,6 +61,8 @@ namespace spades {
 			int GetPing() { return ping; }
 			int GetNumPlayers() { return numPlayers; }
 			int GetMaxPlayers() { return maxPlayers; }
+			bool IsFavorite() { return favorite; }
+			bool SetFavorite(bool favorite) { this->favorite = favorite; }
```

Comment:
`return` is missing in this non-void function. (specified by C++ spec as undefined behavior)